libsweep/CMakeLists.txt

```diff
@@ -0,0 +1,79 @@
+cmake_minimum_required(VERSION 2.8.11)
+project(sweep LANGUAGES C CXX)
+
```

Comment:
Thanks, good catch. I think the explicit `LANGUAGES` is not available in CMake 2.8
https://cmake.org/cmake/help/v2.8.12/cmake.html#command:project